src/features/session/components/Card/Card.tsx

```
@@ -42,50 +42,56 @@ export function Card({
   durationMinutes,
   sessionDate,
 }: CardProps) {
-  // 세션 시작 시간이 이미 지났으면 inProgress 뱃지 우선 표시
-  // statusText가 명시적으로 제공되면 직접 Badge 사용
-  // 그 외에 createdAt이 있으면 RelativeTimeBadge 사용 (hydration 안전)
   const renderStatusBadge = () => {
     if (isPastTime(sessionDate)) {
       return (
-        <Badge radius="max" status="inProgress">
+        <Badge radius="max" status="inProgress" className="text-[10px] px-2 md:text-xs md:px-3">
```

ESLint

[error] Replace `text-[10px]·px-2·md:text-xs·md:px-3` with `px-2·text-[10px]·md:px-3·md:text-xs`

(prettier/prettier)